### ISSUE TITLE
[3.0.0] More schedules configured - issue is that it is executed in reverse order

### ISSUE THREAD

**Issue by bicschneider** [Customatics/leaptest-plugin#18]
```
13:53:28 ....
13:53:28 Schedule  XXX [cec76e11-ec90-4b58-841a-b265fe7ce4e1] successfully detected!
13:53:28 Schedule YYY [3fb945e5-1152-4162-ba84-d6f42c12f265] successfully detected!
13:53:28 The schedule is ready to run as the state is: 'Finished' - let's go
13:53:28 //////////////////////////////////////////////////////////////////////////////////////
13:53:28 Schedule YYY [3fb945e5-1152-4162-ba84-d6f42c12f265] Launched Successfully!
13:53:29 RunId: 'f20e1222-7b63-48c6-b74d-3be73a0f64a2' is not ready yet - wait a minute
```

**Comment by bicschneider** [comment]
Status?

**Comment by romanchetto** [comment]
Hello @bicschneider, I will investigate it. If is found, it will be fixed in the next release

**Comment by romanchetto** [comment]
Hello @bicschneider , Fixed in 3.1.1. I hope it will help you.
